### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by schteve**
Was reading through the docs and code and noticed a few typos, figured I'd fix as I go. Hope this is OK.
Thanks for the very useful crate!

### CODE REVIEW

**Comment by rubberduck203** [comment]
It is more than ok!  
Thank you very much!  
Spelling isn’t my strong suit. 

I’m glad you’ve found the crate useful.

File: README.md [hunk 1]
```diff
@@ -13,20 +13,20 @@ It provides a simple, zero-cost, abstraction to clarify the _intent_ of your app
...
 ## Why Switch-HAL? Why not just use raw GPIO?
 
-Did you mean to drive that line high?  
-Or did you mean to _turn that LED off_?  
-Wait a second... is that LED active _high_?  
-Where's the schematic?  
+Did you mean to drive that line high?
+Or did you mean to _turn that LED off_?
+Wait a second... is that LED active _high_?
+Where's the schematic?
```
Comment (rubberduck203) [inline README.md:19]: The two spaces at the end of these lines was intentional. It’s for formatting. They insert a `<br>`

File: README.md [hunk 2]
```diff
@@ -13,20 +13,20 @@ It provides a simple, zero-cost, abstraction to clarify the _intent_ of your app
...
 Okay... cathode is wired to the input line... that means it's active low.
 
-Now repeat this every place in your code where you need to turn that LED on or off.  
-What happens when the hardware changes?  
+Now repeat this every place in your code where you need to turn that LED on or off.
+What happens when the hardware changes?
```
Comment (rubberduck203) [inline README.md:23]: Same. Spaces were intentional.

File: README.md [hunk 3]
```diff
@@ -13,20 +13,20 @@ It provides a simple, zero-cost, abstraction to clarify the _intent_ of your app
...
 Using the raw GPIO to set pins high and low will have you making changes all over your code base.
 
-Wouldn't it be nicer if you only had to think about that once, when you initialize your application,  
-and from then on out, simply called `led.on()` or `led.off()`.  
-Having an abstraction at the proper level reduces cognitive load.  
-Specifying whether a simple peripheral is active high or low in a single place in your application reduces the maintenace burden.
+Wouldn't it be nicer if you only had to think about that once, when you initialize your application,
+and from then on out, simply called `led.on()` or `led.off()`.
+Having an abstraction at the proper level reduces cognitive load.
+Specifying whether a simple peripheral is active high or low in a single place in your application reduces the maintenance burden.
```
Comment (rubberduck203) [inline README.md:29]: 👍

**Review by rubberduck203 — CHANGES_REQUESTED** [review]
Please put the formatting spaces back in, otherwise looks good to me.

File: README.md (hunk 1 above)
Comment (schteve) [reply README.md]: Ah! Sorry about that, I didn't realize they were meaningful. My editor removed them automatically and I didn't think anything of it. Added them back.

File: README.md (hunk 2 above)
Comment (schteve) [reply README.md]: Done

**Review by rubberduck203 — APPROVED** [review]

**Comment by rubberduck203** [comment]
Thanks again! And seriously, I am so glad you’ve found this useful.

**Pull request merged**